eurito_daps/routines/example/openaire_simple_pipeline.py

```diff
@@ -0,0 +1,138 @@
+
+from nesta.production.orms.orm_utils import get_mysql_engine
+
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine
+
+import luigi
+import datetime
+import os
+import logging
+import requests
+import time
+
+class RootTask(luigi.WrapperTask):
+    '''The root task, which collects the supplied parameters and calls the SimpleTask.
+
+    Args:
+        date (datetime): Date used to label the outputs
+        outputType (str): type of record to be extracted from OpenAIRE API. Accepts "software", "datasets", "publications", "ECProjects"
+        production (bool): test mode or production mode
+    '''
+    date = luigi.DateParameter(default=datetime.datetime.today())
+    outputType = luigi.Parameter()
+    production = luigi.BoolParameter(default=False)
+
+    def requires(self):
+        '''Call the task to run before this in the pipeline.'''
+
+        logging.getLogger().setLevel(logging.INFO)
+        return CollectOpenAireTask(date=self.date,
+                          outputType=self.outputType,
+                          test=not self.production)
+
+
+class CollectOpenAireTask(luigi.Task):
+    '''Collects OpenAIRE projects and writes them into MySQL database
+
+    Args:
+        date (datetime): Date used to label the outputs
+        outputType (str): type of record to be extracted from OpenAIRE API. Accepts "software", "datasets", "publications", "ECProjects"
+        test (bool): run a shorter version of the task if in test mode
+    '''
+    date = luigi.DateParameter(default=datetime.datetime.today())
+    outputType = luigi.Parameter()
+    test = luigi.BoolParameter()
+
+    def output(self):
+        '''Points to the output database engine where the task is marked as done.
+        The luigi_table_updates table exists in test and production databases.
+        '''
+        db_config = get_config(os.environ["MYSQLDB"], "mysqldb")
+        db_config["database"] = 'dev' if self.test else 'production'
+        db_config["table"] = "Example <dummy>"  # Note, not a real table
+        update_id = "SimpleTask_{}".format(self.date)
+        return MySqlTarget(update_id=update_id, **db_config)
+
+    def run(self):
+        '''Collects records from OpenAIRE API and stores them into the 'dev' or 'production' database
+        '''
+        baseUrl = 'http://api.openaire.eu/oai_pmh?verb=ListRecords'
+
+        #engine = create_engine('sqlite:///pythonsqlite.db', echo=False)
+
+        db = 'dev' if self.test else 'production'
+        engine = get_mysql_engine("MYSQLDBCONF", 'mysqldb', db)
+
+        Base.metadata.create_all(engine)
+
+        #open DB session
+        Session = sessionmaker(bind=engine)
+        globals.dbsession = Session()
+
+        #open up requests session
+        reqsession = requests.session()
+        reqsession.keep_alive = False
+
+        resumptionToken = 'Not None'
+
+        currentUrl = baseUrl + '&metadataPrefix=oaf&set=' + self.outputType

```

Comment:
better passing parameters in using `requests.get(url, params={'metadataPrefix':'oaf', 'set':output_type})` than building the query string